app/src/main/java/com/kickstarter/libs/utils/ApplicationLifecycleUtil.java

```diff
@@ -53,7 +53,7 @@ public void onActivityResumed(final @NonNull Activity activity) {
       this.client.config()
         .compose(Transformers.pipeApiErrorsTo(this::handleConfigApiError))
         .compose(Transformers.neverError())
-        .subscribe(this.config::config);
+        .subscribe(c -> this.config.config(c));
```

Comment:
is this one of the edge cases that RequireThisCheck doesn't handle or is it just some debuggering